daemon/container.go

```diff
@@ -627,6 +628,14 @@ func (container *Container) Copy(resource string) (io.ReadCloser, error) {
 		if err != nil {
 			return nil, err
 		}
+		var stat os.FileInfo
+		stat, err = os.Stat(m.Source)
+		if err != nil {
+			return nil, err
+		}
+		if err = fileutils.CreateIfNotExists(dest, stat.IsDir()); err != nil {
+			return nil, err
+		}
```

Comment:
> do we need to remove this new file after the tar is done so it doesn't remain after when the copy is done?

yes

> while not part of this PR, do we need to "defer" the unmount stuff below so that upon an error we unmount stuff?

we already do that, the defer is in a few lines above.